thrift/shared.thrift

```diff
@@ -399,6 +399,7 @@ struct WorkflowExecutionInfo {
   120: optional string taskList
   130: optional bool isCron
   140: optional i64 (js.type = "Long") updateTime
+  150: optional map<string, string> partitionConfig
```

Comment:
This type is used by `DescribeWorkflowExecution`. Adding the config to expose it to customers.